README_eng.md

```diff
@@ -0,0 +1,299 @@
+# Description
+
+We cannot sign transactions on the node side, all transactions must be signed on the client side. In the future, there will be changes to the structures of transactions and we will be required to track them. The ability to input any input data should be covered by tests. 
+
+## Enum TransactionOutput.data
+
+Each transaction output must carry a `data` field. This field is the designation of the purpose of the transaction, we must highlight the following at the moment:
+
+* Transfer Tokens or NFT
+* Token issueance
+* Burning token
+* NFT creation
+
+This field must be `Enum`. All of the `TransactionOutput` structure and related types ​​must be derived `Encode` and `Decode`. This will allow SCALE Codec to correctly decode data fields. That there is no confusion due to changes in data places, we must mark each element of the type with the index `#[codec(index = NUM)]`.
+
+And there is a requirement for testing, each version must be covered with tests that are guaranteed to break if someone changes the internal data in any version. 
+
+```rust
+    #[derive(Encode, Decode...)]
+    pub struct TransactionOutput<AccountId> {
+        #[codec(index = 1)]
+        pub(crate) value: Value,
+        #[codec(index = 2)]
+        pub(crate) destination: Destination<AccountId>,
+        #[codec(index = 3)]
+        pub(crate) data: Option<TxData>,
+    }
+
+    #[derive(Encode, Decode...)]
+    pub enum TxData {
+        // TransferToken data to another user. If it is a token, then the token data must also be transferred to the recipient. 
+        #[codec(index = 1)]
+        TransferTokenV1(..),
+        // A new token creation
+        #[codec(index = 2)]
+        TokenIssuanceV1(..),
```

Comment:
Please make all arms look the same. Either TokenTransfer, TokenIssuance, TokenBurn, or TransferToken, IssueToken, BurnToken.